src/nls/root/strings.js

```diff
@@ -536,15 +538,20 @@ define({
     "EXTENSION_MANAGER_REMOVE_ERROR"       : "Unable to remove one or more extensions: {0}. {APP_NAME} will still reload.",
     "EXTENSION_MANAGER_UPDATE"             : "Update Extension",
     "EXTENSION_MANAGER_UPDATE_ERROR"       : "Unable to update one or more extensions: {0}. {APP_NAME} will still reload.",
+    "EXTENSION_MANAGER_DISABLE"            : "Disable Extension",
+    "EXTENSION_MANAGER_DISABLE_ERROR"      : "Unable to disable one or more extensions: {0}. {APP_NAME} will still reload.",
     "MARKED_FOR_REMOVAL"                   : "Marked for removal",
     "UNDO_REMOVE"                          : "Undo",
     "MARKED_FOR_UPDATE"                    : "Marked for update",
     "UNDO_UPDATE"                          : "Undo",
+    "MARKED_FOR_DISABLING"                 : "Marked for disabling",
```

Comment:
@TomMalbran is right. User has to be aware anyways: after extension manager is dismissed, there will be a dialog asking for confirmation and if canceled all extensions marked will be unmarked (and left intact).

So what about `Marked to be disabled` vs `Marked for disabling`? With all the context, do we need to change it in the first place?